extension/content/customPersonaEditor.js

```diff
@@ -263,6 +263,9 @@ var CustomPersonaEditor = {
       case PersonaService.FIREFOX_ID:
  	window.close();
         break;
+      case PersonaService.SEAMONKEY_ID:
+    window.close();
```

Comment:
[Previos line](https://github.com/mozilla/personas-plus/pull/33#discussion-diff-70428007R264) is also bad formatted. Actually file already contains many formatting issues.

Though when I check the file with SublimeText the line 267 is aligned with line 264. I think Github is displaying file wrong here.

Let me prettify whole file after SeaMonkey PR.